api/sso/routes.py

```diff
@@ -115,7 +115,9 @@ def get_token(self):
             roles=session["user"].get("roles") or [],
         )
 
-        redirect_url = Config.ASSESSMENT_POST_LOGIN_URL  # TODO: Remove defaulting to Assessment, instead use return_app
+        redirect_url = (  # TODO: Remove defaulting to Assessment, instead use return_app
+            f"{Config.ASSESSMENT_FRONTEND_HOST}/assess/fund_dashboard"
```

Comment:
```python
from urllib.parse import urljoin
```

```suggestion
            ulrjoin(Config.ASSESSMENT_FRONTEND_HOST, "/assess/fund_dashboard")
```

I think I have a small preference for using the core url libs as they'll handle some edge cases around trailing `/` and things but I don't think it's blocking.

(Also applies to most the changes)

Removing things that don't change based on environment from the config feels like a clearer pattern to follow, great stuff!